federfrei3/address_mover_trucks
# Conversion to cents <-> euros implemented

-> user input for price_per_day in euros, converted to **cents** in the **database**
-> changed all views to have the price_per_days in **euros** in the **views**

How to test:
- Create a new truck + add the price per day -> should see the same number in the view - WITHOUT DIGITS
- Create a new truck -"- WITH DIGITS

No rounding function yet implemented (in case of more than 2 digits)
